packages/effect/test/Schema/Schema/standardSchemaV1.test.ts

```diff
@@ -2,6 +2,7 @@ import type { StandardSchemaV1 } from "@standard-schema/spec"
 import { Context, Effect, ParseResult, Predicate, Schema } from "effect"
 import { assertTrue, deepStrictEqual, strictEqual } from "effect/test/util"
 import { describe, it } from "vitest"
+import { b } from "vitest/dist/chunks/suite.BJU7kdY9.js"
```

Comment:
this seems like an incorrect automatic import

Reply:
Sorry I will fix that